lua/astrocommunity/completion/tabby-nvim/init.lua

```diff
@@ -0,0 +1,12 @@
+return {
+  { "TabbyML/vim-tabby" },
+  config = function()
```

Comment:
Init function, not config.
```suggestion
  init = function()
```